### ISSUE TITLE
https

### ISSUE THREAD

**Issue by atopalovic** [snowzach/doods#61]
Hello, doods is GREAT, I use it for some time and then I updated my home assistant to a more secure https. After that started my problems. I tried to enable tls but without success. What should I change to make it work? I added to config.yaml URL HTTPS instead of HTTP, 
and under doods config I added under server:
  tls: true
  certfile: /ssl/fullchain.pem
  keyfile: /ssl/privkey.pem

doods log looks like:
2021-03-14T23:33:57.192Z	INFO	server/server.go:284	API Listening	{"package": "server", "address": ":8080", "tls": false, "version": "v0.2.6-0-gb2a1c53-dirty"}

Please help, what do I do wrong?

**Comment by snowzach** [comment]
If you're running DOODS in docker, that's probably not your issue. Are you running Hass.IO?

**Comment by Male20** [comment]
I have exact same issue. Looks like there is a bug somewhere as config file in homeassitant addon is ignored. tls is set to true but log file states false. I updated settings via terminal directly on the file and copied certificate to shared location as doods docker container is not linked to ssl location (normally used by duckdns). Configuration file changes I made in yaml file are ignored same for options.json.  

I have run additional check, looks like putting additional options in server section(via home assistant editor) is not finding way to options.json but changes in doods.detectors: sections are applied. Not sure why this is happening.


![image](https://user-images.githubusercontent.com/77641288/115882968-31322280-a445-11eb-99cb-a059155e7db7.png)
![image](https://user-images.githubusercontent.com/77641288/115883015-3b542100-a445-11eb-9b72-824b2685e813.png)

I found a solution to this problem. In docker in environment I added following entries:
![image](https://user-images.githubusercontent.com/77641288/116315612-ac2e6c80-a7a8-11eb-830a-a2ffb22a8822.png)

Once that was done problem went away. Use your duckdns address and set port forwarding correctly and job done. 
In you config.yaml make sure that address finishes with a port as additional "/' at the end will prevent loading doods

Hassio side and settings are not showing this correctly which is still a bug in my opinion.
